testing/endtoend/endtoend_test.go

```diff
@@ -516,6 +518,33 @@ func (r *testRunner) defaultEndToEndRun() error {
 		}
 	}
 
+	if r.config.TestExecutionRequests && params.ElectraEnabled() {
+		// Test Consolidation Transactions
+		r.comHandler.txGen.SetTxType(eth1.ConsolidationTx)
+		// Wait For an epoch before running evaluator
+		secondsPerEpoch := time.Duration(params.BeaconConfig().SlotsPerEpoch.Mul(params.BeaconConfig().SecondsPerSlot))
+		waitForSync := secondsPerEpoch * time.Second
+		time.Sleep(waitForSync)
```

Comment:
is this the only way to go about it? why shouldn't we be running this in a separate go routine or reading some logs to figure out  when to trigger.